README.md

```diff
@@ -134,6 +134,8 @@ additional details on each available environment variable.
 | `ECS_SHARED_VOLUME_MATCH_FULL_CONFIG` | `true` | When `true`, ECS Agent will compare name, driver options, and labels to make sure volumes are identical. When `false`, Agent will short circuit shared volume comparison if the names match. This is the default Docker behavior. If a volume is shared across instances, this should be set to `false`. | `false` | `false`|
 | `ECS_CONTAINER_INSTANCE_PROPAGATE_TAGS_FROM` | `ec2_instance` | If `ec2_instance` is specified, existing tags defined on the container instance will be registered to Amazon ECS and will be discoverable using the `ListTagsForResource` API. Using this requires that the IAM role associated with the container instance have the `ec2:DescribeTags` action allowed. | `none` | `none` |
 | `ECS_CONTAINER_INSTANCE_TAGS` | `{"tag_key": "tag_val"}` | The metadata that you apply to the container instance to help you categorize and organize them. Each tag consists of a key and an optional value, both of which you define. Tag keys can have a maximum character length of 128 characters, and tag values can have a maximum length of 256 characters. If tags also exist on your container instance that are propagated using the `ECS_CONTAINER_INSTANCE_PROPAGATE_TAGS_FROM` parameter, those tags will be overwritten by the tags specified using `ECS_CONTAINER_INSTANCE_TAGS`. | `{}` | `{}` |
+| `ECS_NVIDIA_RUNTIME` | ecs-nvidia | The Nvidia Runtime to be used to pass Nvidia GPU devices to containers. | ecs-nvidia | Not Applicable |
```

Comment:
Is it worth calling this `ECS_GPU_RUNTIME` instead to be slightly more generic?

Reply:
I don't have a strong opinion.  If you prefer to keep it tied to Nvidia, I'm comfortable with that.

Reply:
left it as is, since gpu support is tied to Nvidia.